src/docs/event.yaml

```diff
@@ -188,6 +189,50 @@ paths:
                 500:
                     description: Internal server error
     /api/event/:id:
+        get:
+          tags: 
+          - "Events"
+          summary: "Get event by id"
+          description: "Returns a single event"
+          operationId: "getEventById"
+          produces:
+          - "application/json"
+          parameters:
+            - name: "id"
+              in: "path"
+              description: "ID of the event to return"
+              required: true
+              type: "string"
+              format: "int64"
+          responses:
+            "200":
```

Comment:
The response object is not showing the swagger UI. This response should look as follows in order for it to work: 
```
        "200":
          description: "An event by spesific id"
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Event"
```